Source/JavaScriptCore/wasm/WasmTypeDefinition.cpp

```diff
@@ -160,7 +160,10 @@ void Projection::dump(PrintStream& out) const
 {
     out.print("(");
     CommaPrinter comma;
-    TypeInformation::get(recursionGroup()).dump(out);
+    if (isPlaceholder())
+        out.print("<current-rec-group>");
```

Comment:
Note: I'm hoping to more comprehensively improve type printing (for bug https://bugs.webkit.org/show_bug.cgi?id=247746) later. This change was the minimal change needed to make the tests work for now.